packages/client/src/pages/CarbonHackCards/ForecastCard/ForecastLineChart.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import ReactApexChart from 'react-apexcharts'
+
+export class ForecastLineChart extends React.Component<
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  { data: object; categories: object },
+  { series: object[]; options: object }
```

Comment:
These types can be a little more specific or just be `any` if they're expected props of apex charts